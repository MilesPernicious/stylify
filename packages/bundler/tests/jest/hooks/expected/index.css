/* File content changed during build */
.a{
	color: red
}
