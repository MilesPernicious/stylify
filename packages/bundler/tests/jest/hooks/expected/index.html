<!-- File content changed during build. -->
<div class="a"></div>
